src/cdn_definitions/schema.yaml

```diff
@@ -34,6 +238,11 @@ definitions:
       $ref: "#/definitions/path_alias"
     uniqueItems: true
 
+  symlink_exception:
+    type: array
+    items:
+      type: string
+      pattern: "^\\/*"
```

Comment:
This pattern doesn't seem right to me, though perhaps the YAML layer is confusing me.

- Firstly, "/" isn't a special character in regexes, so isn't escaping it unnecessary?  Does dropping the "\\" make any difference?
- Secondly, "/*" means "zero or more instances of /" but this doesn't seem useful since it'll match against anything at all.

I guess the intent was to ensure it's an absolute path but I think this is not the correct pattern.  Examples:

```python
# Matches any random string
>>> re.search(r"\/*", 'quux')
<re.Match object; span=(0, 0), match=''>

# Backslash doesn't matter
>>> re.search(r"/*", 'quux')
<re.Match object; span=(0, 0), match=''>

# Maybe it was supposed to be this?
>>> re.search(r"/.*", 'quux')
```